R/promptAndRun.R

```diff
@@ -1,3 +1,19 @@
+#' promptAndRun
```

Comment:
added explanation and also export this function such that it can be used from R directly